tests/sch/__snapshots__/kicad-theme-demo.snap.svg

```diff
@@ -1,8 +1,8 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="1200" height="600" style="background-color: rgb(245, 241, 237)" data-real-to-screen-transform="matrix(65.9340659341,0,0,-65.9340659341,423.0920010989,303.2967032967)"><style>
```

Comment:
regression? The component body should not be transparent.

Also it's pretty hard to read these snapshots on Github now, and we use these snapshots everywhere. Can you figure out a way to make it work without making snapshots hard to read? You can always make the background a parameter and we can set it to "transparent" on tscircuit.com etc. The default should be readable on github.